### PR TITLE
Enable some application properties input via env vars or command line args

### PR DESCRIPTION
**Pull request by v-kerimov**
# Description

Enables supplying such app properties as github.login, github.token, github.api.url, license_filepath and license_conflict with env vars GITHUB_LOGIN, GITHUB_TOKEN, GITHUB_API_URL, LICENSE_FILEPATH and LICENSE_CONFLICT respectively.
Adds clean error logs in case any of the necessary github properties is not supplied.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Addmittance of the env vars by the application was tested manually on a local computer.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by o-kopysov** [comment]
Please take into account this PR https://github.com/Samsung/LPVS/pull/37

**Comment by o-kopysov** [comment]
@v-kerimov  please check the merge conflict and resolve it.

File: README.md
Comment (o-kopysov) [inline README.md:94]: @v-kerimov  In this line we also can set all required variables, not only GITHUB_TOKEN

Comment (v-kerimov) [reply README.md:94]: Yes, but this is made clear earlier in README.md in the lines 75-76.

Btw, wouldn't it be better to modify all the GITHUB and LICENSE env vars by adding LPVS_ prefix to them, so that the they had fewer chances to intersect with an env var used by some other programme?

Comment (o-kopysov) [reply README.md:94]: Yes, agree to rename variables

File: src/main/java/com/lpvs/service/GitHubService.java
Comment (t25kim) [inline src/main/java/com/lpvs/service/GitHubService.java:63]: `GITHUB_LOGIN` and `GITHUB_API_URL` are not mandatory.

File: README.md
Comment (o-kopysov) [inline README.md:88]: @v-kerimov  Incorrect command.
-D flag should be added before JAR file name

java -jar  -Dgithub.login=<`my-login`> -Dgithub.token=<`my-token`> etc.  lpvs-1.0.0.jar

File: README.md
Comment (o-kopysov) [inline README.md:76]: Also please add env variables names inro application.properties as a comment (#) before the names of variables in the file (users should understand the correspondence between env variable name and LPVS variable name).
Example:

# GitHub settings

# Corresponding env. variable is LPVS_GITHUB_LOGIN
github.login=

File: src/main/java/com/lpvs/service/GitHubService.java
Comment (o-kopysov) [inline src/main/java/com/lpvs/service/GitHubService.java:47]: @v-kerimov 
Please test the Docker case.
When I run the command:
```
docker run -p 7896:7896 --name lpvs -e LPVS_GITHUB_TOKEN="uiuyhggjhg77676ghhg" lpvs:latest
```
the log is:
```
2022-09-09 11:50:24.675  INFO 1 --- [           main] com.lpvs.LicensePreValidationSystem      : Starting LicensePreValidationSystem v1.0.0 using Java 11.0.16 on 475de6038553 with PID 1 (/LPVS/target/lpvs-1.0.0.jar started by root in /LPVS/target)
2022-09-09 11:50:24.680  INFO 1 --- [           main] com.lpvs.LicensePreValidationSystem      : No active profile set, falling back to 1 default profile: "default"
2022-09-09 11:50:28.088  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 7896 (http)
2022-09-09 11:50:28.131  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2022-09-09 11:50:28.133  INFO 1 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.65]
2022-09-09 11:50:28.404  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2022-09-09 11:50:28.404  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 3519 ms
2022-09-09 11:50:28.660 ERROR 1 --- [           main] com.lpvs.service.GitHubService           : LPVS_GITHUB_TOKEN(github.token) is not set.
```

File: src/main/java/com/lpvs/service/GitHubService.java
Comment (o-kopysov) [inline src/main/java/com/lpvs/service/GitHubService.java:56]: @v-kerimov  Please remove unnecessary println

**Review by o-kopysov — APPROVED** [review]
LGTM

File: src/main/resources/application.properties
Comment (t25kim) [inline src/main/resources/application.properties:24]: Please check indentations.

```suggestion
#Corresponding env. variable LPVS_LICENSE_FILEPATH
license_filepath=classes/licenses.json

# Source of the license conflicts information: json or scanner
#Corresponding env. variable LPVS_LICENSE_CONFLICT
license_conflict=json

# GitHub settings
# Corresponding env. variable LPVS_GITHUB_LOGIN
github.login=
# Corresponding env. variable LPVS_GITHUB_TOKEN
github.token=
# Corresponding env. variable LPVS_GITHUB_API_URL
github.api.url=
# Corresponding env. variable LPVS_GITHUB_SECRET
```

**Comment by o-kopysov** [comment]
Hm... Locally I have no issues with the build for PR.

**Comment by o-kopysov** [comment]
@MoonkiHong  @t25kim 
I've finally fixed the build for this PR. Please take a look at it.

**Review by tdrozdovsky — APPROVED** [review]

**Review by t25kim — APPROVED** [review]
LGTM Thanks!

**Review by MoonkiHong — APPROVED** [review]
All review comments are correctly applied. +1

**Comment by MoonkiHong** [comment]
@v-kerimov Before merging this PR, we need your `sign-off`. Please ask @o-kopysov to amend your commit to adhere your signature (with email) properly. 🙏 So that we could merge your PR.

**Pull request merged**
